adc/microphone_adc/README.adoc

```diff
@@ -20,7 +20,7 @@ image::microphone_adc_plotter.png[]
 
 Wiring up the device requires 3 jumpers, to connect VCC (3.3v), GND, and AOUT. The example here uses ADC0, which is GP26. Power is supplied from the 3.3V pin.
 
-WARNING: Most boards will take a range of voltages from the Pico's default 3.3v to the 5 volts commonly seen on other microcontrollers. Ensure your board works with either the 3.3V or 5V supplied by the Pico.
+WARNING: Most boards will take a range of voltages from the Pico's default 3.3v to the 5 volts commonly seen on other microcontrollers. Ensure your board works with either the 3.3V or 5V supplied by the Pico. If you supply 5V, then the board must have a level shifter to reduce the output voltage to 3.3V to be compatible with the Pico's GPIOs, otherwise you risk damaging the Pico!
```

Comment:
I'm not sure this added sentence is actually that helpful as IIRC level shifters are used for translating digital signals, not constantly-varying analogue signals like we're talking about with this ADC example?

Perhaps it makes sense to just say something like:
"WARNING: Most boards will take a range of VCC voltages from the Pico's default 3.3V to the 5 volts commonly seen on other microcontrollers. Ensure your board doesn't output an analogue signal greater than 3.3V as this may result in permanent damage to the Pico's ADC."

WDYT @aallan ?

Reply:
> level shifters are used for translating digital signals, not constantly-varying analogue signals like we're talking about with this ADC example

You are correct, a level shifter won't help you here.

> Perhaps it makes sense to just say something like:
"WARNING: Most boards will take a range of VCC voltages from the Pico's default 3.3V to the 5 volts commonly seen on other microcontrollers. Ensure your board doesn't output an analogue signal greater than 3.3V as this may result in permanent damage to the Pico's ADC."

That sounds okay to me!

Reply:
Oh...right 🤦